src/ImageSharp/ColorSpaces/Conversion/ColorSpaceConverter.Adapt.cs

```diff
@@ -0,0 +1,198 @@
+﻿// <copyright file="ColorSpaceConverter.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.ColorSpaces.Conversion
+{
+    using System;
+
+    using ImageSharp.ColorSpaces;
+    using ImageSharp.ColorSpaces.Conversion.Implementation.Rgb;
+
+    /// <content>
+    /// Performs chromatic adaptation on the various color spaces.
+    /// </content>
+    public partial class ColorSpaceConverter
```

Comment:
I'm assuming you mean bulk operations etc? I can internalize this for now. It's not something that is explicitly required for the moment.